### PR TITLE
fix: alter shared source fresh schema will make it non-shared

### LINKED ISSUE #19799: Alter shared source will make it non-shared
### Describe the bug

This caused recent flaky CI for `e2e_test/source_inline/kafka/protobuf/alter_source_shared.slt`

From diagnose info, I found `mv_user_more` uses `Source`, instead `SourceBackfill`, unlike `mv_user_2` and `mv_user`.
```

--- Actor Traces ---
>> Actor 291
Actor 291: `CREATE SOURCE src_user INCLUDE timestamp WITH (connector = 'kafka', properties.bootstrap.server = 'message_queue:29092', topic = 'pb_alter_source_shared_test', scan.startup.mode = 'earliest') FORMAT PLAIN ENCODE PROTOBUF (schema.registry = 'http://schemaregistry:8082', message = 'test.User')` [100.008s]
  Epoch 7653331424182272 [229.995ms]
    Source 12300002712 [229.995ms]
      receive_barrier [229.995ms]

>> Actor 292
Actor 292: `CREATE SOURCE src_user INCLUDE timestamp WITH (connector = 'kafka', properties.bootstrap.server = 'message_queue:29092', topic = 'pb_alter_source_shared_test', scan.startup.mode = 'earliest') FORMAT PLAIN ENCODE PROTOBUF (schema.registry = 'http://schemaregistry:8082', message = 'test.User')` [100.008s]
  Epoch 7653331424182272 [229.995ms]
    Source 12400002712 [229.995ms]
      receive_barrier [229.995ms]

>> Actor 293
Actor 293: `CREATE SOURCE src_user INCLUDE timestamp WITH (connector = 'kafka', properties.bootstrap.server = 'message_queue:29092', topic = 'pb_alter_source_shared_test', scan.startup.mode = 'earliest') FORMAT PLAIN ENCODE PROTOBUF (schema.registry = 'http://schemaregistry:8082', message = 'test.User')` [100.008s]
  Epoch 7653331424182272 [229.995ms]
    Source 12500002712 [229.995ms]
      receive_barrier [229.995ms]

>> Actor 294
Actor 294: `CREATE SOURCE src_user INCLUDE timestamp WITH (connector = 'kafka', properties.bootstrap.server = 'message_queue:29092', topic = 'pb_alter_source_shared_test', scan.startup.mode = 'earliest') FORMAT PLAIN ENCODE PROTOBUF (schema.registry = 'http://schemaregistry:8082', message = 'test.User')` [100.008s]
  Epoch 7653331424182272 [229.995ms]
    Source 12600002712 [229.995ms]
      receive_barrier [229.995ms]

>> Actor 315
Actor 315: `CREATE MATERIALIZED VIEW mv_user AS SELECT * FROM src_user` [99.568s]
  Epoch 7653331424182272 [229.995ms]
    Materialize 13B00000003 [229.995ms]
      Project 13B00000002 [229.995ms]
        RowIdGen 13B00000001 [229.995ms]
          SourceBackfill 13B00002727 [229.995ms]
            Merge 13B00000000 [229.995ms]

>> Actor 316
Actor 316: `CREATE MATERIALIZED VIEW mv_user AS SELECT * FROM src_user` [99.568s]
  Epoch 7653331424182272 [229.995ms]
    Materialize 13C00000003 [229.995ms]
      Project 13C00000002 [229.995ms]
        RowIdGen 13C00000001 [229.995ms]
          SourceBackfill 13C00002727 [229.995ms]
            Merge 13C00000000 [229.995ms]

>> Actor 317
Actor 317: `CREATE MATERIALIZED VIEW mv_user AS SELECT * FROM src_user` [99.568s]
  Epoch 7653331424182272 [229.995ms]
    Materialize 13D00000003 [229.995ms]
      Project 13D00000002 [229.995ms]
        RowIdGen 13D00000001 [229.995ms]
          SourceBackfill 13D00002727 [229.995ms]
            Merge 13D00000000 [229.995ms]

>> Actor 318
Actor 318: `CREATE MATERIALIZED VIEW mv_user AS SELECT * FROM src_user` [99.568s]
  Epoch 7653331424182272 [229.995ms]
    Materialize 13E00000003 [229.995ms]
      Project 13E00000002 [229.995ms]
        RowIdGen 13E00000001 [229.995ms]
          SourceBackfill 13E00002727 [229.995ms]
            Merge 13E00000000 [229.995ms]

>> Actor 347
Actor 347: `CREATE MATERIALIZED VIEW mv_user_2 AS SELECT * FROM src_user` [98.908s]
  Epoch 7653331424182272 [229.995ms]
    Materialize 15B00000003 [229.995ms]
      Project 15B00000002 [229.995ms]
        RowIdGen 15B00000001 [229.995ms]
          SourceBackfill 15B00002727 [229.995ms]
            Merge 15B00000000 [229.995ms]

>> Actor 348
Actor 348: `CREATE MATERIALIZED VIEW mv_user_2 AS SELECT * FROM src_user` [98.908s]
  Epoch 7653331424182272 [229.995ms]
    Materialize 15C00000003 [229.995ms]
      Project 15C00000002 [229.995ms]
        RowIdGen 15C00000001 [229.995ms]
          SourceBackfill 15C00002727 [229.995ms]
            Merge 15C00000000 [229.995ms]

>> Actor 349
Actor 349: `CREATE MATERIALIZED VIEW mv_user_2 AS SELECT * FROM src_user` [98.908s]
  Epoch 7653331424182272 [229.995ms]
    Materialize 15D00000003 [229.995ms]
      Project 15D00000002 [229.995ms]
        RowIdGen 15D00000001 [229.995ms]
          SourceBackfill 15D00002727 [229.995ms]
            Merge 15D00000000 [229.995ms]

>> Actor 350
Actor 350: `CREATE MATERIALIZED VIEW mv_user_2 AS SELECT * FROM src_user` [98.908s]
  Epoch 7653331424182272 [229.995ms]
    Materialize 15E00000003 [229.995ms]
      Project 15E00000002 [229.995ms]
        RowIdGen 15E00000001 [229.995ms]
          SourceBackfill 15E00002727 [229.995ms]
            Merge 15E00000000 [229.995ms]

>> Actor 551
Actor 551: `CREATE TABLE s1 (v1 STRUCT<v2 INT>, t1 TIMESTAMP, z1 timestamptz, c1 CHARACTER VARYING) WITH (connector = 'datagen', fields.v1.v2.kind = 'random', fields.v1.v2.min = '1', fields.v1.v2.max = '100', fields.v1.v2.seed = '1', fields.t1.kind = 'random', fields.t1.max_past = '2h 37min', fields.t1.max_past_mode = 'relative', fields.t1.seed = '3', fields.z1.kind = 'random', fields.z1.max_past = '2h 37min', fields.z1.max_past_mode = 'relative', fields.z1.seed = '3', fields.c1.kind = 'random', fields.c1.length = '100', fields.c1.seed = '3', datagen.rows.per.second = '10', datagen.split.num = '5') FORMAT PLAIN ENCODE JSON` [94.038s]
  Epoch 7653331424182272 [229.995ms]
    Materialize 22700000006 [229.995ms]
      RowIdGen 22700000005 [229.995ms]
        Union 22700000004 [229.995ms]
          Merge 22700000001 [229.995ms]
          Merge 22700000003 [229.995ms]

>> Actor 552
Actor 552: `CREATE TABLE s1 (v1 STRUCT<v2 INT>, t1 TIMESTAMP, z1 timestamptz, c1 CHARACTER VARYING) WITH (connector = 'datagen', fields.v1.v2.kind = 'random', fields.v1.v2.min = '1', fields.v1.v2.max = '100', fields.v1.v2.seed = '1', fields.t1.kind = 'random', fields.t1.max_past = '2h 37min', fields.t1.max_past_mode = 'relative', fields.t1.seed = '3', fields.z1.kind = 'random', fields.z1.max_past = '2h 37min', fields.z1.max_past_mode = 'relative', fields.z1.seed = '3', fields.c1.kind = 'random', fields.c1.length = '100', fields.c1.seed = '3', datagen.rows.per.second = '10', datagen.split.num = '5') FORMAT PLAIN ENCODE JSON` [94.038s]
  Epoch 7653331424182272 [229.995ms]
    Materialize 22800000006 [229.995ms]
      RowIdGen 22800000005 [229.995ms]
        Union 22800000004 [229.995ms]
          Merge 22800000001 [229.995ms]
          Merge 22800000003 [229.995ms]

>> Actor 553
Actor 553: `CREATE TABLE s1 (v1 STRUCT<v2 INT>, t1 TIMESTAMP, z1 timestamptz, c1 CHARACTER VARYING) WITH (connector = 'datagen', fields.v1.v2.kind = 'random', fields.v1.v2.min = '1', fields.v1.v2.max = '100', fields.v1.v2.seed = '1', fields.t1.kind = 'random', fields.t1.max_past = '2h 37min', fields.t1.max_past_mode = 'relative', fields.t1.seed = '3', fields.z1.kind = 'random', fields.z1.max_past = '2h 37min', fields.z1.max_past_mode = 'relative', fields.z1.seed = '3', fields.c1.kind = 'random', fields.c1.length = '100', fields.c1.seed = '3', datagen.rows.per.second = '10', datagen.split.num = '5') FORMAT PLAIN ENCODE JSON` [94.038s]
  Epoch 7653331424182272 [229.995ms]
    Materialize 22900000006 [229.995ms]
      RowIdGen 22900000005 [229.995ms]
        Union 22900000004 [229.995ms]
          Merge 22900000001 [229.995ms]
          Merge 22900000003 [229.995ms]

>> Actor 554
Actor 554: `CREATE TABLE s1 (v1 STRUCT<v2 INT>, t1 TIMESTAMP, z1 timestamptz, c1 CHARACTER VARYING) WITH (connector = 'datagen', fields.v1.v2.kind = 'random', fields.v1.v2.min = '1', fields.v1.v2.max = '100', fields.v1.v2.seed = '1', fields.t1.kind = 'random', fields.t1.max_past = '2h 37min', fields.t1.max_past_mode = 'relative', fields.t1.seed = '3', fields.z1.kind = 'random', fields.z1.max_past = '2h 37min', fields.z1.max_past_mode = 'relative', fields.z1.seed = '3', fields.c1.kind = 'random', fields.c1.length = '100', fields.c1.seed = '3', datagen.rows.per.second = '10', datagen.split.num = '5') FORMAT PLAIN ENCODE JSON` [94.038s]
  Epoch 7653331424182272 [229.995ms]
    Materialize 22A00000006 [229.995ms]
      RowIdGen 22A00000005 [229.995ms]
        Union 22A00000004 [229.995ms]
          Merge 22A00000003 [229.995ms]
          Merge 22A00000001 [229.995ms]

>> Actor 555
Actor 555: `CREATE TABLE s1 (v1 STRUCT<v2 INT>, t1 TIMESTAMP, z1 timestamptz, c1 CHARACTER VARYING) WITH (connector = 'datagen', fields.v1.v2.kind = 'random', fields.v1.v2.min = '1', fields.v1.v2.max = '100', fields.v1.v2.seed = '1', fields.t1.kind = 'random', fields.t1.max_past = '2h 37min', fields.t1.max_past_mode = 'relative', fields.t1.seed = '3', fields.z1.kind = 'random', fields.z1.max_past = '2h 37min', fields.z1.max_past_mode = 'relative', fields.z1.seed = '3', fields.c1.kind = 'random', fields.c1.length = '100', fields.c1.seed = '3', datagen.rows.per.second = '10', datagen.split.num = '5') FORMAT PLAIN ENCODE JSON` [94.038s]
  Epoch 7653331424182272 [229.995ms]
    Source 22B00002716 [229.995ms]
      receive_barrier [229.995ms]

>> Actor 556
Actor 556: `CREATE TABLE s1 (v1 STRUCT<v2 INT>, t1 TIMESTAMP, z1 timestamptz, c1 CHARACTER VARYING) WITH (connector = 'datagen', fields.v1.v2.kind = 'random', fields.v1.v2.min = '1', fields.v1.v2.max = '100', fields.v1.v2.seed = '1', fields.t1.kind = 'random', fields.t1.max_past = '2h 37min', fields.t1.max_past_mode = 'relative', fields.t1.seed = '3', fields.z1.kind = 'random', fields.z1.max_past = '2h 37min', fields.z1.max_past_mode = 'relative', fields.z1.seed = '3', fields.c1.kind = 'random', fields.c1.length = '100', fields.c1.seed = '3', datagen.rows.per.second = '10', datagen.split.num = '5') FORMAT PLAIN ENCODE JSON` [94.038s]
  Epoch 7653331424182272 [229.995ms]
    Source 22C00002716 [229.995ms]
      receive_barrier [229.995ms]

>> Actor 557
Actor 557: `CREATE TABLE s1 (v1 STRUCT<v2 INT>, t1 TIMESTAMP, z1 timestamptz, c1 CHARACTER VARYING) WITH (connector = 'datagen', fields.v1.v2.kind = 'random', fields.v1.v2.min = '1', fields.v1.v2.max = '100', fields.v1.v2.seed = '1', fields.t1.kind = 'random', fields.t1.max_past = '2h 37min', fields.t1.max_past_mode = 'relative', fields.t1.seed = '3', fields.z1.kind = 'random', fields.z1.max_past = '2h 37min', fields.z1.max_past_mode = 'relative', fields.z1.seed = '3', fields.c1.kind = 'random', fields.c1.length = '100', fields.c1.seed = '3', datagen.rows.per.second = '10', datagen.split.num = '5') FORMAT PLAIN ENCODE JSON` [94.038s]
  Epoch 7653331424182272 [229.995ms]
    Source 22D00002716 [229.995ms]
      receive_barrier [229.995ms]

>> Actor 558
Actor 558: `CREATE TABLE s1 (v1 STRUCT<v2 INT>, t1 TIMESTAMP, z1 timestamptz, c1 CHARACTER VARYING) WITH (connector = 'datagen', fields.v1.v2.kind = 'random', fields.v1.v2.min = '1', fields.v1.v2.max = '100', fields.v1.v2.seed = '1', fields.t1.kind = 'random', fields.t1.max_past = '2h 37min', fields.t1.max_past_mode = 'relative', fields.t1.seed = '3', fields.z1.kind = 'random', fields.z1.max_past = '2h 37min', fields.z1.max_past_mode = 'relative', fields.z1.seed = '3', fields.c1.kind = 'random', fields.c1.length = '100', fields.c1.seed = '3', datagen.rows.per.second = '10', datagen.split.num = '5') FORMAT PLAIN ENCODE JSON` [94.038s]
  Epoch 7653331424182272 [229.995ms]
    Source 22E00002716 [229.995ms]
      receive_barrier [229.995ms]

>> Actor 559
Actor 559: `CREATE TABLE s1 (v1 STRUCT<v2 INT>, t1 TIMESTAMP, z1 timestamptz, c1 CHARACTER VARYING) WITH (connector = 'datagen', fields.v1.v2.kind = 'random', fields.v1.v2.min = '1', fields.v1.v2.max = '100', fields.v1.v2.seed = '1', fields.t1.kind = 'random', fields.t1.max_past = '2h 37min', fields.t1.max_past_mode = 'relative', fields.t1.seed = '3', fields.z1.kind = 'random', fields.z1.max_past = '2h 37min', fields.z1.max_past_mode = 'relative', fields.z1.seed = '3', fields.c1.kind = 'random', fields.c1.length = '100', fields.c1.seed = '3', datagen.rows.per.second = '10', datagen.split.num = '5') FORMAT PLAIN ENCODE JSON` [94.038s]
  Epoch 7653331424182272 [229.995ms]
    Dml 22F00000002 [229.995ms]
      Source 22F00002719 [229.995ms]

>> Actor 560
Actor 560: `CREATE TABLE s1 (v1 STRUCT<v2 INT>, t1 TIMESTAMP, z1 timestamptz, c1 CHARACTER VARYING) WITH (connector = 'datagen', fields.v1.v2.kind = 'random', fields.v1.v2.min = '1', fields.v1.v2.max = '100', fields.v1.v2.seed = '1', fields.t1.kind = 'random', fields.t1.max_past = '2h 37min', fields.t1.max_past_mode = 'relative', fields.t1.seed = '3', fields.z1.kind = 'random', fields.z1.max_past = '2h 37min', fields.z1.max_past_mode = 'relative', fields.z1.seed = '3', fields.c1.kind = 'random', fields.c1.length = '100', fields.c1.seed = '3', datagen.rows.per.second = '10', datagen.split.num = '5') FORMAT PLAIN ENCODE JSON` [94.038s]
  Epoch 7653331424182272 [229.995ms]
    Dml 23000000002 [229.995ms]
      Source 23000002719 [229.995ms]

>> Actor 561
Actor 561: `CREATE TABLE s1 (v1 STRUCT<v2 INT>, t1 TIMESTAMP, z1 timestamptz, c1 CHARACTER VARYING) WITH (connector = 'datagen', fields.v1.v2.kind = 'random', fields.v1.v2.min = '1', fields.v1.v2.max = '100', fields.v1.v2.seed = '1', fields.t1.kind = 'random', fields.t1.max_past = '2h 37min', fields.t1.max_past_mode = 'relative', fields.t1.seed = '3', fields.z1.kind = 'random', fields.z1.max_past = '2h 37min', fields.z1.max_past_mode = 'relative', fields.z1.seed = '3', fields.c1.kind = 'random', fields.c1.length = '100', fields.c1.seed = '3', datagen.rows.per.second = '10', datagen.split.num = '5') FORMAT PLAIN ENCODE JSON` [94.038s]
  Epoch 7653331424182272 [229.995ms]
    Dml 23100000002 [229.995ms]
      Source 23100002719 [229.995ms]

>> Actor 562
Actor 562: `CREATE TABLE s1 (v1 STRUCT<v2 INT>, t1 TIMESTAMP, z1 timestamptz, c1 CHARACTER VARYING) WITH (connector = 'datagen', fields.v1.v2.kind = 'random', fields.v1.v2.min = '1', fields.v1.v2.max = '100', fields.v1.v2.seed = '1', fields.t1.kind = 'random', fields.t1.max_past = '2h 37min', fields.t1.max_past_mode = 'relative', fields.t1.seed = '3', fields.z1.kind = 'random', fields.z1.max_past = '2h 37min', fields.z1.max_past_mode = 'relative', fields.z1.seed = '3', fields.c1.kind = 'random', fields.c1.length = '100', fields.c1.seed = '3', datagen.rows.per.second = '10', datagen.split.num = '5') FORMAT PLAIN ENCODE JSON` [94.038s]
  Epoch 7653331424182272 [229.995ms]
    Dml 23200000002 [229.995ms]
      Source 23200002719 [229.995ms]

>> Actor 635
Actor 635: `CREATE MATERIALIZED VIEW mv_user_more AS SELECT * FROM src_user` [91.978s]
  Epoch 7653331424182272 [229.995ms]
    Materialize 27B00000003 [229.995ms]
      Project 27B00000002 [229.995ms]
        RowIdGen 27B00000001 [229.995ms]
          Source 27B00002727 [229.995ms]
            receive_barrier [229.995ms]

>> Actor 636
Actor 636: `CREATE MATERIALIZED VIEW mv_user_more AS SELECT * FROM src_user` [91.978s]
  Epoch 7653331424182272 [229.995ms]
    Materialize 27C00000003 [229.995ms]
      Project 27C00000002 [229.995ms]
        RowIdGen 27C00000001 [229.995ms]
          Source 27C00002727 [229.995ms]
            receive_barrier [229.995ms]

>> Actor 637
Actor 637: `CREATE MATERIALIZED VIEW mv_user_more AS SELECT * FROM src_user` [91.978s]
  Epoch 7653331424182272 [229.995ms]
    Materialize 27D00000003 [229.995ms]
      Project 27D00000002 [229.995ms]
        RowIdGen 27D00000001 [229.995ms]
          Source 27D00002727 [229.995ms]
            receive_barrier [229.995ms]

>> Actor 638
Actor 638: `CREATE MATERIALIZED VIEW mv_user_more AS SELECT * FROM src_user` [91.978s]
  Epoch 7653331424182272 [229.995ms]
    Materialize 27E00000003 [229.995ms]
      Project 27E00000002 [229.995ms]
        RowIdGen 27E00000001 [229.995ms]
          Source 27E00002727 [229.995ms]
            receive_barrier [229.995ms]


```

### Error message/log

_No response_

### To Reproduce

_No response_

### Expected behavior

_No response_

### How did you deploy RisingWave?

_No response_

### The version of RisingWave

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by xxchan**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://raw.githubusercontent.com/risingwavelabs/risingwave/17af8a747593ebdbfa826691daf75bdab7d14fa0/.github/contributor-license-agreement.txt).

## What's changed and what's your intention?
fix #19799

Note: This problem only exists for `REFRESH SCHEMA` (i.e., source with schema regitry), but not for `ADD COLUMN`, because only the former updated `source_info`

## Checklist

- [ ] I have written necessary rustdoc comments.
- [ ]  I have added necessary unit tests and integration tests.
- [ ]  I have added test labels as necessary. 
- [ ]  I have added fuzzing tests or opened an issue to track them. 
- [ ]  My PR contains breaking changes. 
- [ ]  My PR changes performance-critical code, so I will run (micro) benchmarks and present the results. 
- [ ]  My PR contains critical fixes that are necessary to be merged into the latest release. 

## Documentation

- [ ]  My PR needs documentation updates. 

<details>
<summary><b>Release note</b></summary>

</details>

### CODE REVIEW

**Comment by xxchan** [comment]
* **#19803** <a href="https://app.graphite.dev/github/pr/risingwavelabs/risingwave/19803?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#19802** <a href="https://app.graphite.dev/github/pr/risingwavelabs/risingwave/19802?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/risingwavelabs/risingwave/19802?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`


<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: src/frontend/src/handler/create_source.rs [hunk 1]
```diff
@@ -302,12 +302,71 @@ fn get_name_strategy_or_default(name_strategy: Option<AstString>) -> Result<Opti
...
+    session: &SessionImpl,
+    format_encode: &FormatEncodeOptions,
+    with_properties: Either<&WithOptions, &WithOptionsSecResolved>,
+    create_source_type: CreateSourceType,
+) -> Result<(Option<Vec<ColumnCatalog>>, StreamSourceInfo)> {
+    let (columns_from_resolve_source, mut source_info) =
+        if create_source_type == CreateSourceType::SharedCdc {
+            bind_columns_from_source_for_cdc(session, format_encode)?
+        } else {
+            bind_columns_from_source_for_non_cdc(session, format_encode, with_properties).await?
```
Comment (xxchan) [inline src/frontend/src/handler/create_source.rs:359]: - Renamed old `bind_columns_from_source` to `bind_columns_from_source_for_non_cdc`. Make it private. Previously it created a `mut StreamSourceInfo`
- Add new `bind_columns_from_source`, and put the code modifying `StreamSourceInfo` inside.

File: src/frontend/src/handler/create_source.rs [hunk 2]
```diff
@@ -1748,26 +1806,14 @@ pub async fn handle_create_source(
...
-        source_info.is_distributed = !create_cdc_source_job;
-    }
+    let create_source_type = CreateSourceType::from_with_properties(&session, &*with_properties);
+    let (columns_from_resolve_source, source_info) = bind_columns_from_source(
+        &session,
+        &format_encode,
+        Either::Left(&with_properties),
+        create_source_type,
+    )
+    .await?;
```
Comment (xxchan) [inline src/frontend/src/handler/create_source.rs:1816]: Wrapping logic into `bind_columns_from_source`

**Review by stdrc — APPROVED** [review]
LGTM

File: src/frontend/src/handler/create_source.rs [hunk 3]
```diff
@@ -302,12 +302,71 @@ fn get_name_strategy_or_default(name_strategy: Option<AstString>) -> Result<Opti
...
+    ) -> Self {
+        if with_properties.is_shareable_cdc_connector() {
+            CreateSourceType::SharedCdc
+        } else if with_properties.is_shareable_non_cdc_connector()
+            && session
+                .env()
+                .streaming_config()
+                .developer
+                .enable_shared_source
+            && session.config().streaming_use_shared_source()
```
Comment (BugenZhao) [inline src/frontend/src/handler/create_source.rs:328]: Is it possible that we use a different streaming or session configuration when refreshing the schema registry compared to when it was initially created? Will this lead to any problem?

Comment (xxchan) [reply src/frontend/src/handler/create_source.rs:328]: Yes, it will lead to problem

**Pull request merged**